indra/newview/llfloaterimsessiontab.cpp

```diff
@@ -433,16 +434,34 @@ void LLFloaterIMSessionTab::draw()
 
 void LLFloaterIMSessionTab::enableDisableCallBtn()
 {
-    if (LLVoiceClient::instanceExists() && mVoiceButton)
+    if (!mVoiceButton)
+        return;
+
+    BOOL enable = FALSE;
+
+    if (mSessionID.notNull() && mSession && mSession->mSessionInitialized && mSession->mCallBackEnabled)
     {
-        mVoiceButton->setEnabled(
-            mSessionID.notNull()
-            && mSession
-            && mSession->mSessionInitialized
-            && LLVoiceClient::getInstance()->voiceEnabled()
-            && LLVoiceClient::getInstance()->isVoiceWorking()
-            && mSession->mCallBackEnabled);
```

Comment:
If voiceEnabled() and isVoiceWorking() are somehow insuficient, then LLAvatarActions::canCall() won't work too. Or is it mSessionInitialized that is false?

WebRTC is fresh and has issues, most likely problem is not in LLFloaterIMSessionTab, but likely WebRTC got stuck in a weird state.

Reply:
Agreed.  There is also, in fact, a chance that the issue is server-side.

Reply:
> There is also, in fact, a chance that the issue is server-side.

Issue is that viewer's UI gets stuck in a state where user can't disable outgoing call.

Reply:
> Agreed. There is also, in fact, a chance that the issue is server-side.

The bug described in the issue https://github.com/secondlife/viewer/issues/2325 was reproduced on the build version `Atlasaurus 7.1.9.10326512121`

But on the build compiled from the latest source code this bug was not reproduced

There was an other variant of the bug observed: the button became disabled during the early recall,
but it returned to a normal state after hang up made via the button `Cancel` on the outgoing call floater

This bug seems to be 100% UI-related, the voice channel works well with no issues